src/worker/utils/molecule.ts

```diff
@@ -47,6 +47,26 @@ export const get_molecule = (smiles: string, RDKit: RDKitModule) => {
   }
 };
 
+export const get_query_molecule_memory_unsafe = (structure: string, RDKit: RDKitModule) => {
+  const cachedQMolecule = getJSMolFromCache(structure);
+  if (cachedQMolecule) return cachedQMolecule;
+  if (!structure) return null;
+  if (!RDKit) return null;
+  const qmol = RDKit.get_qmol(structure);
+  storeJSMolInCache(structure, qmol);
+  return qmol;
+};
+
+export const get_query_molecule = (structure: string, RDKit: RDKitModule) => {
```

Comment:
is this being used? did you mean to use it in convertMolNotation?

Reply:
Now used in `convertMolNotation`